ui/raidboss/data/07-dt/raid/r5s.txt

```diff
@@ -72,7 +72,7 @@ hideall "--sync--"
 174.8 "Get Down! (Echo)" #Ability { id: "A765", source: "Dancing Green" }
 
 # Debuff and Dodge Dance
-182.2 "Let's Dance! (Cleave) 1" Ability { id: ["A76D", "A76E"], source: "Dancing Green" }
+182.2 "Let's Dance! (Cleave) 1" Ability { id: ["A76D", "A76E"], source: "Dancing Green" } window 10,1.5
 184.8 "Let's Dance! (Cleave) 2" #Ability { id: "A76D", source: "Dancing Green" }
```

Comment:
I think for consistency with the 2nd set later, we should just drop the sync on the first cleave here instead?